app/assets/stylesheets/_main.scss

```diff
@@ -51,6 +51,13 @@ p {
   background-color: white;
 }
 
+.bops-headers {
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+  background-color: govuk-colour("white");
+}
+
```

Comment:
I think this will conflict with the new assessment tasks view. (I can double check shortly.)

Reply:
Just spoke with Fede and the header needs to sticky